### DISCUSSION TITLE
API Authentication For Azure

### DISCUSSION THREAD

**Discussion by hbc-acai** [apache/airflow#41644] [Q&A]
Hello, 

I am wondering if anyone is aware of an Azure Auth Backend implementation that I can look at.  

Someone asked a similar question on StackOverflow but did not get an answer. 
https://stackoverflow.com/questions/75850782/azure-ad-authentication-for-airflow-rest-api

Thank you

**Comment by seniut** [comment]
@hbc-acai  hello. Have you found some solution? I cannot find any good examples
